docs/index.rst

```diff
@@ -17,6 +17,7 @@ Welcome to xaitk-saliency's documentation!
    design
    faq
    miscellaneous
+   api_docs
```

Comment:
I agree @Purg but maybe between the Installation and Review Process sections?

Reply:
^Yes, I think what Mary suggests makes sense to me. We probably want to keep the Installation section before API docs since that is what people usually try to find first in order to see how hard/easy it is to get started.